docs/quickstart/csharp.md

```diff
@@ -215,18 +217,19 @@ Note that you may also have to change the permissions of the protoc and protobuf
 binaries in the `Grpc.Tools` package under `examples/csharp/helloworld/packages`
 to executable in order to run the commands below.
 
-From the `examples/csharp/helloworld` directory:
+From the `examples/csharp/helloworld` directory, or the `examples/csharp/helloworld-from-cli` directory if using the .NET Core SDK:
 
 **Windows**
 
 ```
-> packages/Grpc.Tools.1.0.0/tools/windows_x86/protoc -I../../protos --csharp_out Greeter --grpc_out Greeter ../../protos/helloworld.proto --plugin=protoc-gen-grpc=packages/Grpc.Tools.1.0.0/tools/windows_x86/grpc_csharp_plugin.exe
+> ./generate_protos.bat
 ```
 
 **Linux (or OS X by using macosx_x64 directory)**
 
 ```
-$ packages/Grpc.Tools.1.0.0/tools/linux_x64/protoc -I../../protos --csharp_out Greeter --grpc_out Greeter ../../protos/helloworld.proto --plugin=protoc-gen-grpc=packages/Grpc.Tools.1.0.0/tools/linux_x64/grpc_csharp_plugin
+$ GRPC_TOOLS_PACKAGE="Grpc.Tools.1.0.0"
```

Comment:
I didn't mean this type of variable. I meant the `{{ site.data.config.grpc_release_branch }}` so that we don't have to remember to bump the version number each release.